Source/Filter/BilinearFilter.cpp

```diff
@@ -56,20 +56,21 @@ namespace {
 
 void BilinearFilter::ComputeStencils(){
     WARPX_PROFILE("BilinearFilter::ComputeStencils()");
-    stencil_length_each_dir = npass_each_dir;
+    for (int i = 0; i < AMREX_SPACEDIM; ++i)
```

Comment:
I added a new member to `amrex::IntVect` so you can do:
```suggestion
    stencil_length_each_dir = npass_each_dir::toArray<unsigned int>();
```
https://github.com/AMReX-Codes/amrex/pull/1768

Reply:
Thanks a lot!